src/mpid/ch4/netmod/ofi/ofi_impl.h

```diff
@@ -351,6 +351,16 @@ MPL_STATIC_INLINE_PREFIX fi_addr_t MPIDI_OFI_av_to_phys(MPIDI_av_entry_t * av)
     }
 }
 
+MPL_STATIC_INLINE_PREFIX fi_addr_t MPIDI_OFI_av_vni_to_phys(MPIDI_av_entry_t * av, int vni_idx)
```

Comment:
I'd rather this patch be included in a series where it is used. Adding it here is kind of a non-sequitur.

Reply:
Makes sense. I can drop it from this PR.